catalog/app/containers/Bucket/requests.js

```diff
@@ -340,18 +287,18 @@ const WORKFLOWS_CONFIG_PATH = '.quilt/workflows/config.yml'
 export const workflowsList = async ({ s3, bucket }) => {
   try {
     const response = await fetchFile({ s3, bucket, path: WORKFLOWS_CONFIG_PATH })
-    return parseWorkflows(response.Body.toString('utf-8'))
+    return workflows.parse(response.Body.toString('utf-8'))
   } catch (e) {
     if (e instanceof errors.FileNotFound || e instanceof errors.VersionNotFound)
-      return emptyWorkflowsConfig
+      return workflows.emptyConfig
```

Comment:
this return doesnt seem necessary, bc we already have the same statement at the end of the function

Reply:
It's an "early exit":
If error is `FileNotFound` or `VersionNotFound`, then early exit with `emptyConfig`. Otherwise, **log exception and** exit with `emptyConfig`